coreneuron/CMakeLists.txt

```diff
@@ -126,11 +126,16 @@ if(CORENRN_ENABLE_GPU)
   # (https://cmake.org/cmake/help/latest/policy/CMP0119.html), so using a single .cu file and
   # setting LANGUAGE=CXX in non-GPU builds does not work.
   list(APPEND CORENEURON_CODE_FILES ${CORENEURON_CUDA_FILES})
-  if(CORENRN_ENABLE_OPENMP_OFFLOAD)
+  if(CORENRN_ACCELERATOR_OFFLOAD STREQUAL "OpenMP")
```

Comment:
I was considering to implement it like this but thought that above option would be equivalent. Thanks for noticing this didn't actually produce the correct code.